### PR TITLE
Hastic instance name

### LINKED ISSUE #547: Webhooks: Hastic instance name / url
Now you don't know which Hastic instance is sending you a webhook
Though it can be critical in cases like `Analytics is down`
We should add a field with some hastic-server identifier

### PR DESCRIPTION
**Pull request by amper43**
fixes #547 

changes:
- added `instanceName` field to server info and webhook payload
- added label `instanceName` to `prometheus-hastic-exporter`

### CODE REVIEW

File: server/src/services/notification_service.ts [hunk 1]
```diff
@@ -1,9 +1,10 @@
 import { Segment } from '../models/segment_model';
 import * as AnalyticUnit from '../models/analytic_unit_model';
-import { HASTIC_WEBHOOK_URL, HASTIC_WEBHOOK_TYPE, HASTIC_WEBHOOK_SECRET } from '../config';
+import { HASTIC_WEBHOOK_URL, HASTIC_WEBHOOK_TYPE, HASTIC_WEBHOOK_SECRET, HASTIC_INSTANCE_NAME } from '../config';
 
 import axios from 'axios';
 import * as querystring from 'querystring';
+import * as _ from 'lodash';
```
Comment (rozetko) [inline server/src/services/notification_service.ts]: Why do we need `lodash` here?

File: server/src/config.ts [hunk 2]
```diff
@@ -35,6 +35,7 @@ export const INSIDE_DOCKER = process.env.INSIDE_DOCKER !== undefined;
 export const PRODUCTION_MODE = process.env.NODE_ENV !== 'development';
 
 export const ZMQ_CONNECTION_STRING = createZMQConnectionString();
+export const HASTIC_INSTANCE_NAME = getConfigField('HASTIC_INSTANCE_NAME', null);
```
Comment (rozetko) [inline server/src/config.ts]: Maybe we should make default value `'unset'`, for example? Instead of null

Comment (rozetko) [reply server/src/config.ts]: Bu I don't know if it's a good idea
What do you think?

Comment (jonyrock) [reply server/src/config.ts]: or
```
var os = require("os");
os.hostname();
```

Comment (amper43) [reply server/src/config.ts]: @jonyrock hostname often returns `localhost`

File: server/src/services/notification_service.ts (hunk 1 above)
Comment (amper43) [inline server/src/services/notification_service.ts]:
```suggestion
```

File: server/src/config.ts (hunk 2 above)
Comment (jonyrock) [reply server/src/config.ts]: @amper43 still better than `undefined`

Comment (rozetko) [reply server/src/config.ts]: +1 for `os.hostname()`

**Pull request merged**